index.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 const Build = require('./lib/buildmodel');
+const Job = require('./lib/jobmodel');
```

Comment:
Can we please use a better file naming schema here.

Something like:
 - `./lib/models/job'
 - `./lib/job`